src/MultiOwnable.sol

```diff
@@ -96,12 +104,14 @@ contract MultiOwnable {
 
     /// @notice Removes an owner from the given `index`.
     ///
-    /// @dev Reverts if the owner is not registered at `index`.
+    /// @dev Reverts if the provided owner is not registered at `index`.
     ///
     /// @param index The index to remove from.
-    function removeOwnerAtIndex(uint256 index) public virtual onlyOwner {
+    /// @param _owner The owner at the specific index.
+    function removeOwnerAtIndex(uint256 index, bytes calldata _owner) public virtual onlyOwner {
```

Comment:
Style guide [says](https://docs.soliditylang.org/en/latest/style-guide.html#avoiding-naming-collisions) `singleTrailingUnderscore_` for avoiding naming collision.